Sources/iron/App.hx

```diff
@@ -156,20 +156,30 @@ class App {
 		traitInits.remove(f);
 	}
 
-	public static function notifyOnUpdate(f:Void->Void) {
-		traitUpdates.push(f);
+	public static function notifyOnUpdate(f:Void->Void, updatePriority = 100) {
+		traitUpdates.push({priority:updatePriority, f:f});
+		traitUpdates.sort(function(a, b):Int {
```

Comment:
Sorting for every added trait seem a bit like a waste, instead introduce a `traitUpdatesDirty` bool flag and only sort the array once in `update()`?